File: wwwroot/js/requisicao.js
function saveRequisicaoBtn() {
    var requisicaoData = {
        codRequisicao: "0", // ou outro valor apropriado
        pedido: "0",
        posto: "0",
        computador: "0", // ou outro valor apropriado      
        acessoInternet: "2025-01-01",
        internet: "N",
        medico: $('#Medico').val().split(" ", 1),
        guia: "",
        guiaPrincipal: "",
        guiaData: "2025-01-01",
        guiaDataAutorizacao: "2025-01-01",
        guiaSenha: "",
        guiaSenhaValidade: "",
        plano: $('#Plano').val(),
        carteirinha: $('#Carteirinha').val(),
        validadeCarteirinha: $('#ValidadeCarteirinha').val(),
        leito: $('#Leito').val(),
        observacao: $('#Observacao').val(),
        obsMedicamento: $('#Medicamentos').val(),
        previsaoEntrega: "2025-01-01",
        paciente: "0", // ou outro valor apropriado
        contrato: $('#Contrato').val(),
        fatura: "0", // ou outro valor apropriado
        lancamento: "",
        faturamento: "",
        origem: "",
        postosus: "0", // ou outro valor apropriado
        urgente: "N",
        parcial: "N",
        bruto: "0.0", // ou outro valor apropriado
        desconto: "0.0", // ou outro valor apropriado
        acrescimo: "0.0", // ou outro valor apropriado
        coleta: "0.0", // ou outro valor apropriado
        total: "0.0", // ou outro valor apropriado
        pago: "0.0",
        particular: "N",
        usuarioCad: "0", // ou outro valor apropriado
        obsBio: "",
        podeAssinar: "N",
        bloqueado: "N",
        faturavel: "N",
        statusId: "1"
    };
    alert(JSON.stringify(requisicaoData));
    $.ajax({
        url: '/api/requisicao',
        type: 'POST',
        contentType: 'application/json',
        data: JSON.stringify(requisicaoData),
        success: function (response) {
            alert('Requisição salva com sucesso!');
        },
        error: function (xhr, status, error) {
            console.log('Erro na solicitação AJAX:', error);
            console.log('Status:', status);
            console.log('Resposta completa:', xhr);
            // Verificar se a resposta contém um objeto JSON
            var response = xhr.responseJSON || {};
            if (response.errors) {
                console.log('Erros:', response.errors);
            } else {
                console.log('Nenhum erro encontrado na resposta.');
            }
        }
    });

};

File: Models/Requisicao.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Drawing;
using static System.Runtime.InteropServices.JavaScript.JSType;

namespace ilis.Models
{
    public class Requisicao
    {
        [Key]
        public int codRequisicao { get; set; }

        public int pedido { get; set; }
        public int posto { get; set; }
        public int computador { get; set; }  
        public string acessoInternet  { get; set; }
         public string internet { get; set; }
        public string medico  { get; set; }
        public string guia  { get; set; }
        public string guiaPrincipal  { get; set; }
        public string guiaData  { get; set; }
        public string guiaDataAutorizacao  { get; set; }
        public string guiaSenha  { get; set; }
        public string guiaSenhaValidade  { get; set; }
        public string plano  { get; set; }
        public string carteirinha  { get; set; }
        public string validadeCarteirinha  { get; set; }
        public string leito  { get; set; }
        public string observacao  { get; set; }
        public string obsMedicamento  { get; set; }
        public string previsaoEntrega  { get; set; }
        public int paciente { get; set; }
        public int contrato { get; set; }
        public int fatura   { get; set; }
        public string lancamento  { get; set; }
        public string faturamento  { get; set; }
        public string origem { get; set; }
        public int postosus { get; set; }
        public string urgente { get; set; }
        public string parcial { get; set; }
        public double bruto { get; set; }
        public double desconto { get; set; }
        public double acrescimo { get; set; }
        public double coleta { get; set; }
        public double total { get; set; }
        public string pago { get; set; }
        public string particular { get; set; }
        public int usuarioCad { get; set; }
        public string obsBio { get; set; }
        public string podeAssinar { get; set; }
        public string bloqueado { get; set; }
        public string faturavel { get; set; }
        public int statusId { get; set; }

        // Adicione outras propriedades conforme necessário
    }
}